src/cargo-kani/src/call_cbmc_viewer.rs

```diff
@@ -60,7 +60,7 @@ impl KaniSession {
     }
 
     fn cbmc_variant(&self, file: &Path, extra_args: &[&str], output: &Path) -> Result<()> {
-        let mut args = self.cbmc_flags(file)?;
+        let mut args = self.cbmc_flags(file, None)?;
```

Comment:
`cbmc_flags` shouldn't take an option, and we should propagate the harness to this function so it can be passed it, I'm pretty sure.